CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Developer guide
+
+## CI
+
+Checks are automatically run on certain pushed commits for testing and linting
+purposes. Some are defined by [.github/workflows/ci.yaml][] while others are
+configured outside of this repository.
+
+[.github/workflows/ci.yaml]: ./.github/workflows/ci.yaml
+
+## Pre-commit
+
+[pre-commit][] is used for various checks (see [configuration][]).
+
+You can either [install it yourself][] to catch issues before pushing or look
+for the [pre-commit.ci run][] after pushing.
+
+[pre-commit]: https://pre-commit.com/
+[configuration]: ./.pre-commit-config.yaml
+[install it yourself]: https://pre-commit.com/#install
+[pre-commit.ci run]: https://results.pre-commit.ci/repo/github/493877605
+
+## Snakemake formatting
+
+We use [`snakefmt`](https://github.com/snakemake/snakefmt) to ensure consistency in style across Snakemake files in this project.
+
+### Installing
+
+- Using mamba/bioconda:
```

Comment:
I think we're back to recommending conda (not mamba) - more context in this [slack thread](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1701901812441119)

Reply:
Good to know about conda/mamba! fixed with https://github.com/nextstrain/zika/pull/28/commits/8cf73262fe266286460c83ea851f9d7e2a9a2b46